apps/src/javalab/JavalabView.jsx

```diff
@@ -145,48 +150,20 @@ class JavalabView extends React.Component {
   }
 }
 
-// We use the UnconnectedJavalabView to make this component's methods testable.
-// This is a deprecated pattern but calling shallow().dive().instance() on the
-// connected JavalabView does not give us access to the methods owned by JavalabView.
-export const UnconnectedJavalabView = JavalabView;
-export default connect(
-  state => ({
-    isProjectLevel: state.pageConstants.isProjectLevel,
-    isReadOnlyWorkspace: state.pageConstants.isReadOnlyWorkspace,
-    channelId: state.pageConstants.channelId,
-    isDarkMode: state.javalab.isDarkMode
-  }),
-  dispatch => ({
-    appendOutputLog: log => dispatch(appendOutputLog(log)),
-    setIsDarkMode: isDarkMode => dispatch(setIsDarkMode(isDarkMode))
-  })
-)(UnconnectedJavalabView);
-
 const styles = {
```

Comment:
(Ignore if this is still a WIP)
It looks like some of these styles may have been unintentionally changed while resolving merge conflicts.
![image](https://user-images.githubusercontent.com/8324574/117491238-c85aa800-af24-11eb-896c-aaded8b231ce.png)
Which leads to an interesting view
![image](https://user-images.githubusercontent.com/8324574/117491300-e0cac280-af24-11eb-9b1a-f6abca8ae328.png)

Reply:
🤦‍♀️Fixing now. Thank you